### PR TITLE
BookA-Pset_GeoObsBoreholeWaterLevel

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_GeoObsBoreholeWaterLevel

## Description: 

Properties for describing water levels

| Property | Datatype | Description |
|----------|----------|-------------|
|WaterLevel|IfcLengthMeasure|measured distance to water level in borehole from ground surface, in meters|
|IsPiezometric|IfcBoolean|Defines whether hight measurements are made vs the piezometric surface (true) or the surface of the groundwater (false).|

### CODE REVIEW

File: IFC4x3/Properties/w/WaterLevelType_1c5OF998b1_8SQzyRB0_vU/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Type of measured water level (free groundwater / rise to this measured level).
```
Comment (SergejMuhic) [inline IFC4x3/Properties/w/WaterLevelType_1c5OF998b1_8SQzyRB0_vU/Documentation.md]: Having trouble understanding this description. What would be an example of a `WaterLevelType`?

Cannot make suggestion if I do not understand.

Comment (larswik) [reply IFC4x3/Properties/w/WaterLevelType_1c5OF998b1_8SQzyRB0_vU/Documentation.md]: > Having trouble understanding this description. What would be an example of a `WaterLevelType`?
> 
> Cannot make suggestion if I do not understand.

Ping @j-weil.

Comment (j-weil) [reply IFC4x3/Properties/w/WaterLevelType_1c5OF998b1_8SQzyRB0_vU/Documentation.md]: Water level types would be
"free" or "piezometric / confined" when the groundwater body is limited on top by a seal and the water will rise in a pipe that sticks into it. Helpful?

Comment (larswik) [reply IFC4x3/Properties/w/WaterLevelType_1c5OF998b1_8SQzyRB0_vU/Documentation.md]: proposal: IsPiezometric : IfcBoolean - Defines whether hight measurements are made vs the piezometric surface (true) or the surface of the groundwater (false)

Found this definition of piezometric surface: "the imaginary surface to which groundwater rises. under hydrostatic pressure in wells or spring."

@j-weil: Please confirm!

Comment (larswik) [reply IFC4x3/Properties/w/WaterLevelType_1c5OF998b1_8SQzyRB0_vU/Documentation.md]: Fixed in [1b1bc72](https://github.com/bSI-InfraRoom/IFC-Specification/pull/585/commits/1b1bc72aa23d64541fd411ac8250a25a69c46a2d)

**Comment by larswik** [comment]
Also saw that WaterLevelMbgS now should be just WaterLevel. Fixed in [1b1bc72](https://github.com/bSI-InfraRoom/IFC-Specification/pull/585/commits/1b1bc72aa23d64541fd411ac8250a25a69c46a2d)

File: IFC4x3/Properties/i/IsPiezometric_1c5OF998b1_8SQzyRB0_vU/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Defines whether hight measurements are made vs the piezometric surface (true) or the surface of the groundwater (false).
```
Comment (SergejMuhic) [inline IFC4x3/Properties/i/IsPiezometric_1c5OF998b1_8SQzyRB0_vU/Documentation.md]: Typo carried over from Excel. 😃 
```suggestion
Defines whether height measurements are made vs the piezometric surface (true) or the surface of the groundwater (false).
```

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**